src/nni_manager/training_service/pai/paiK8S/paiK8STrainingService.ts

```diff
@@ -100,12 +100,15 @@ class PAIK8STrainingService extends PAITrainingService {
         }
     }
     
-    //TODO: update trial parameters
+    // update trial parameters for multi-phase
     public async updateTrialJob(trialJobId: string, form: TrialJobApplicationForm): Promise<TrialJobDetail> {
-        const trialJobDetail: undefined | TrialJobDetail = this.trialJobsMap.get(trialJobId);
+        const trialJobDetail: PAITrialJobDetail | undefined = this.trialJobsMap.get(trialJobId);
         if (trialJobDetail === undefined) {
             throw new Error(`updateTrialJob failed: ${trialJobId} not found`);
         }
+        // Write file content ( parameter.cfg ) to working folders
+        await this.writeParameterFile(trialJobDetail.logPath, form.hyperParameters);
```

Comment:
why write to logPath instead of trail job's working directory?

Reply:
This path will synchronize log files generated in docker container.

Reply:
then is `trialjob.workingDirectory` useful now?

Reply:
In PAITrainingService, there are two directories. One is workingDirectory, this directory stores nniManager logs and datastore data, the default path is located in `~/nni/experiments/{experimentId}`. Another path is `logPath`, which could synchronize data with docker container, store output data from trial and parameter files, the default path is located in `{nniManagerNFSMountPath}/{experimentId}/{trialId}`. The parameter.cfg file should be put in `logPath` folder instead of `workingDirectory `.

Reply:
@SparkSnail so why not merge the two directory together? according to my understanding in local mode there is only one directory right?

Reply:
As discussed offline, NNI maintains workingDirectory and logPath in PAI mode. This structure maybe need refactor, this pr will keep current logic temporarily.